docs/guides/debugging-with-vscode.md

```diff
@@ -0,0 +1,88 @@
+# Debugging with vscode
+
+[Visual Studo Code](https://code.visualstudio.com/) is a light weight editor, with excellent debugging support. In this guide you'll be shown on how to get debugging working with `fuse-box` on both web apps and node applications. The source code for this example application can be found [here](/andrew-ross/fuse-debug).
+
+The source code structure should look as follows:
+```
+src +	
+  - client +
+    - components +
+      - Adder.tsx
+      - Main.tsx
+      - ServerCaps.tsx
+    - index.tsx
+  - server +
+     - index.ts
+```
+
+## Browser Debugging
+
+Make sure the [debugger for chrome](https://marketplace.visualstudio.com/items?itemName=msjsdiag.debugger-for-chrome) is installed. 
+Then Start up fuse box by running `npm start` in the terminal and wait for the `Listening @ http://localhost:3000` message. In vscode open the debug panel and select the `Launch Chrome` configuration from the drop down and launch by pressing `f5`.
+
+Add a break point to either the `render` or `add` function in the `Adder.tsx` and hit the Click Me button. It should break as demoed below:
+
+![debugging gif](../images/debugging.gif)
```

Comment:
also here, full link otherwise it won't be visible on the website

Reply:
Done